src/app/header/Header.tsx

```diff
@@ -1,24 +1,37 @@
-import React from 'react';
-// import StyledHeader from './Header.styles';
-
+import React from "react";
+import { Box, Flex, HStack, Spacer, Text } from "@chakra-ui/react";
+import { SearchBar } from "./components/SearchBar";
+import { UserButton } from "./components/UserButton";
+import { TermButtons } from "./components/TermButtons";
 
 export interface HeaderProps {
   /**
-   * Content
-   * Subject to change
+   * setTerm: changes the state of the term selected with the buttons
    */
-  content?: string;
+  setTerm: React.Dispatch<React.SetStateAction<string>>;
 }
 
 /**
  * Primary UI component for content
  */
-export const Header: React.FC<HeaderProps> = ({
-  content
-}) => {
+export const Header: React.FC<HeaderProps> = ({ setTerm }) => {
   return (
-    <div>
-      {content}
-    </div>
-  )
-};
\ No newline at end of file
+    <Box bg="tomato" h={66} px="10" py="4">
+      <Flex minW="900px">
+        {/* TODO: turn this into a logo */}
+        <Box w={225} textAlign="center">
+          <Text fontSize="x-large" color="white" fontWeight="bold">
+            clockwork
+          </Text>
+        </Box>
+        <Spacer />
```

Comment:
As said in the Discord, I don't think these spacer elements would be good to use in the header.